src/pages/database/Database.tsx

```diff
@@ -16,12 +30,31 @@ const Database: React.FC<DatabaseProps> = props => {
     <>
       <div>
         {props.notes.map((note: Note, index: number) => (
-          <Link to={`/note/${note.id}`} key={index}>
-            <div className="database-note">
-              <p>Title: {note.title}</p>
-              <p>Latest Update: {note.latestUpdate}</p>
-            </div>
-          </Link>
+          <ContextMenuTrigger id={note.id} holdToDisplay={1000}>
+            <Link to={`/note/${note.id}`} key={index}>
+              <div className="database-note">
+                <p>Title: {note.title}</p>
+                <p>Latest Update: {note.latestUpdate}</p>
+              </div>
+            </Link>
+            <ContextMenu id={note.id}>
+              <Menu vertical>
+                <MenuItem onClick={() => {}}>
+                  <Menu.Item onClick={() => props.deleteNoteHandler(note.id)}>
+                    <Icon name="trash alternate" />
+                    Delete Note
+                  </Menu.Item>
+                  <Menu.Item onClick={() => props.createNoteHandler()}>
+                    <Icon name="add" />
+                    Add Note
+                  </Menu.Item>
+                  <RenameNotePopup
+                    {...{ note: note, updateNoteTitleHandler: props.updateNoteTitleHandler }}
+                  />
+                </MenuItem>
+              </Menu>
+            </ContextMenu>
```

Comment:
same as per the comment for the AllDatabases page

i think we can put this part into the generalised context menu component too (if we are going that route)